.pre-commit-config.yaml

```diff
@@ -84,8 +84,10 @@ repos:
           - flake8-return
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: "v0.1.15"
+    rev: "v0.2.0"
     hooks:
+      - id: ruff-format
+        args: ["--preview"]
```

Comment:
Shouldn't this happen after the step below? Automated fixes could impact formatting

Reply:
good point so maybe we could have chicken/egg problem in case of long line...

but yes let's do it as you suggest and we can adjust later if we see any issue in future